serde-tests/json.rs

```diff
@@ -0,0 +1,132 @@
+use pretty_assertions::assert_eq;
+use serde_json::json;
+
+use super::AllTypes;
+
+use bson::{doc, Bson, JavaScriptCodeWithScope, OwnedRawBson, RawArrayBuf, RawDocumentBuf};
+
+use serde::{Deserialize, Serialize};
+
+#[test]
+fn all_types_json() {
```

Comment:
this test was moved without modification from `test.rs`. The one after it is new though.